test/discrete_test.json

```diff
@@ -1376,6 +1376,33 @@
     }
   ], 
   [
+    "Poisson(0.0)", 
```

Comment:
In that case I don't understand how to add in a new test? I can only find there, and test/discrete_test.lst, but that gives no information on what the values should be.